docs/src/pages/landing-page/QuasarComponents.vue

```diff
@@ -109,29 +110,27 @@ export default defineComponent({
 </script>
 
 <style lang="scss" scoped>
-$number-of-card-columns-gt-sm: 5;
-$number-of-card-columns-sm-max: 3;
-$number-of-card-columns-xs-max: 1;
 
 .components {
   display: grid;
   letter-spacing: 3px;
-  grid-template-columns: repeat($number-of-card-columns-gt-sm, 1fr);
+  grid-template-columns: repeat(auto-fit, minmax(200px, 1fr));
   gap: 24px;
-  margin: 240px 64px 100px 64px;
+  margin: 48px 84px 100px 84px;
 
-  @media screen and (max-width: $breakpoint-sm-max) {
-    margin: auto;
-    grid-template-columns: repeat($number-of-card-columns-sm-max, 1fr);
-    gap: 20px;
+  @media screen and (min-width: $breakpoint-xs-max) {
+    grid-template-columns: repeat(auto-fit, minmax(300px, 1fr));
   }
-  @media screen and (max-width: $breakpoint-xs-max) {
-    margin: auto;
-    grid-template-columns: repeat($number-of-card-columns-xs-max, 1fr);
-    gap: 20px;
+
+  &__card {
```

Comment:
What is this needed for?

Reply:
This fixes the issue with top border left/right radii. I have udpated to use the helper class instead.